client/src/main/java/io/split/client/impressions/strategy/ProcessImpressionMTK.java

```diff
@@ -0,0 +1,25 @@
+package io.split.client.impressions.strategy;
+
+import io.split.client.impressions.Impression;
+import io.split.client.impressions.ImpressionsResult;
+import io.split.client.impressions.UniqueKeysTracker;
+
+import java.util.List;
+
+public class ProcessImpressionMTK implements ProcessImpressionStrategy{
```

Comment:
Please rename this to `ProcessImpressionNone`. 

We have 3 strategies: `debug`, `optimized` and `none` so this request is to keep the naming convention.

Reply:
I changed the name.